serial/uart.c

```diff
@@ -206,7 +208,9 @@ uart0_rx_intr_handler(void *para)
   if (UART_RXFIFO_FULL_INT_ST == (READ_PERI_REG(UART_INT_ST(uart_no)) & UART_RXFIFO_FULL_INT_ST)
   ||  UART_RXFIFO_TOUT_INT_ST == (READ_PERI_REG(UART_INT_ST(uart_no)) & UART_RXFIFO_TOUT_INT_ST))
   {
-    //os_printf("stat:%02X",*(uint8 *)UART_INT_ENA(uart_no));
+#ifdef UART_DBG
+    os_printf("stat:%02X",*(uint8 *)UART_INT_ENA(uart_no));
```

Comment:
This is a noisy debug printf. By uncommenting you prevent the error printf (see above) to be enabled without also getting this noisy debug printf...